os/arch/arm/src/artik05x/src/artik055_alc5658.c

```diff
@@ -296,19 +295,8 @@ int s5j_alc5658_initialize(int minor)
 			goto errout_with_irq;
 		}
 
-		/* No we can embed the ALC5658/I2C/I2S conglomerate into a PCM decoder
-		 * instance so that we will have a PCM front end for the the ALC5658
-		 * driver.
-		 */
-
-		pcm = pcm_decode_initialize(alc5658);
-		if (!pcm) {
-			auddbg("ERROR: Failed create the PCM decoder\n");
-			ret = -ENODEV;
-			goto errout_with_alc5658;
-		}
 
-		ret = audio_register(devname, pcm);
+		ret = audio_register(devname, alc5658);
```

Comment:
Then we can delete pcm_decoder.c, can't we?

Reply:
Yes. I will remove it.